src/components/Modal/ModalConfirmSwap.tsx

```diff
@@ -32,7 +32,7 @@ import { NearBlock, NearTX, QueueTransactions } from "@src/types/interfaces"
 export interface ModalConfirmSwapPayload extends CallRequestIntentProps {}
 
 const ModalConfirmSwap = () => {
-  const [transactionQueue, setTransactionQueue] = useState(0)
+  const [transactionQueue, setTransactionQueue] = useState(1)
```

Comment:
Is it okay that default is 1 ?

Reply:
Yes, it is totally OK cause optimistically we know that transactionQueue will be at lease 1.